src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2253,6 +2253,103 @@ public function get_test_authorization_header() {
 		return $result;
 	}
 
+	/**
+	 * Tests persistent_object_cache health check.
```

Comment:
This is not really an appropriate documentation :)

Please describe in one or two sentences what this really does, not just the name of the test it refers to, which is a bit pointless.